lib/liftoff/xcodeproj_helper.rb

```diff
@@ -81,10 +83,18 @@ def target
       @target ||= ObjectPicker.choose_item('target', available_targets)
     end
 
+    def test_target
+      @test_target ||= ObjectPicker.choose_item('test target', available_test_targets)
+    end
+
     def available_targets
       xcode_project.targets.to_a.reject { |t| t.name.end_with?('Tests') }
     end
 
+    def available_targets
+      xcode_project.targets.to_a.select { |t| t.name.end_with?('Tests') }
```

Comment:
Is it worth extracting `xcode_project.targets.to_a`, since you use it twice? Maybe call it `targets`?